tools/push-aur.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+if [ -z $CI ]; then
+    echo "Only run in CI"
+    exit 1
+fi
+
+cd "${0%/*}/.."
+ROOT="$(pwd)"
+TMP="${ROOT}/.tmp"
+
+sudo pacman -S --noconfirm git openssh
+
+PACKAGES="$(cd .tmp/pkgs; ls -1)"
+for PKG in $PACKAGES; do
+    if ! grep -Fxq $PKG "$ROOT"/aur/packages; then
+        continue
+    fi
+    echo "Updating AUR package $PKG ..."
+    cd "$TMP"/aur
+    git clone -n ssh://aur.archlinux.org/$PKG.git
+    cp -a "$ROOT"/packages/$PKG .
+    cd $PKG
+
+    # generate .SRCINFO
+    sudo -Eu nobody makepkg --printsrcinfo > .SRCINFO
+
+    # update .gitignore
+    cat "$ROOT"/aur/gitignore-base >> .gitignore
+
+    # commit and push
+    git add .
```

Comment:
I think we have to be careful here. In this repo some files in `/packages/*/` are excluded by the top-level `.gitignore`, but the AUR package repo might not have a proper `.gitignore` to exclude them as well.

Reply:
That's why I'm appending the contents of `gitignore-base` above.

Anyway, what is the common practice for `.gitignore`s in the AUR? I often see maintainers using negated ignores to selectively include files and ignore everything else. Maybe we could generate the `.gitignore` from scratch here using `git ls-files packages/$PKG/` and a static list of common ignores.

Reply:
> That's why I'm appending the contents of gitignore-base above.

Ah, I overlooked that.

> Anyway, what is the common practice for .gitignores in the AUR?

I don't know. I usually just exclude:

    src/
    pkg/
    *.pkg.tar.xz
    *.pkg.tar.zst
    <package>.*.tar.gz